compiler/damlc/daml-stdlib-src/DA/Tuple.daml

```diff
@@ -56,3 +56,8 @@ curry3 f a b c = f (a,b,c)
 -- | Converts a curried function to a function on a triple.
 uncurry3 : (a -> b -> c -> d) -> ((a, b, c) -> d)
 uncurry3 f ~(a,b,c) = f a b c
+
+instance Functor ((,,) a b) where
```

Comment:
I'm not a big fan of the `Functor` instances for tuples. I'd say having it on pairs was a mistake. Let's not double down on this mistake and thus _not_ add this instance for triples.

Reply:
Being a marginal fan would do :)

I don't see the harm in having it, but anyways - I hear you, Martin -  you don't want it, so it's gone. Only tests remain in the PR.